src/components/PageHeader/PageHeader.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Link, StaticQuery, graphql } from 'gatsby'
+import { StaticQuery, graphql } from 'gatsby'
 
-const PageHeader = ({ title, link }) => {
+const PageHeader = () => {
   return (
     <StaticQuery
       query={graphql`
```

Comment:
Can we convert this to a similar pattern that is used in the Header Component? See [here](https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/src/components/Header/Header.js#L20) removing the inlined styles. Effectively removing the StaticQuery Component and render prop, in favor of the useStaticQuery hook.

1. Please convert the [header](https://github.com/Vets-Who-Code/vets-who-code-app/blob/master/src/components/PageHeader/PageHeader.js#L20) element over to GatsbyBackgrounImage.
2. Let's convert this over to a fluid image query instead of a fixed image query.